definitions/features/service.rb

```diff
@@ -96,17 +100,39 @@ def allowed_action?(action)
   end
 
   def filter_services(service_list, options)
+    service_list = include_unregistered_services(service_list, options[:include])
     if options[:only] && options[:only].any?
       service_list = service_list.select do |service|
         options[:only].any? { |opt| service.matches?(opt) }
       end
+      service_list = include_unregistered_services(service_list, options[:only])
     end
+
     if options[:exclude] && options[:exclude].any?
       service_list = service_list.reject { |service| options[:exclude].include?(service.name) }
     end
     service_list
   end
 
+  def include_unregistered_services(service_list, services_filter)
+    return service_list unless services_filter
+    return service_list unless services_filter.any?
+
+    services_filter = services_filter.reject do |obj|
+      service_list.any? { |service| service.matches?(obj) }
+    end
+
+    unregistered_service_list = services_filter.map do |obj|
+      if obj.is_a? String
+        system_service(obj)
+      elsif valid_sys_service?(obj)
```

Comment:
We don't do type check in other places so it is probably unnecessary. In the reject above we expect Service type or string anyway.

Reply:
In the reject above, it matches string/Service type against existing service_list of available features.

For crond service, after maintenance -mode status command, foreman-maintain suggests procedure to fix maintenance-mode for crond i.e  either to stop/start.
I have used argument as `{ :only => system_service('crond') }` in this case that wouldn't be part of existing service_list of available features. That's why added type checking here.